src/libraries/System.Drawing.Primitives/README.md

```diff
@@ -0,0 +1,16 @@
+# System.Drawing.Primitives
+This assembly exposes fundamental drawing APIs such as [`Point`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.point) and [`Size`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.size) which don't require access to native GDI+ libraries.
+
+Note that in contrast to `System.Drawing.Common`, `System.Drawing.Primitives` supports not just Windows, but all officially supported platforms.
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/system.drawing.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [ ] [We don't accept refactoring changes due to new language features](../README.md#secondary-bars)
```

Comment:
This is confusing. The bar in the linked readme is "we don't accept refactoring changes" by default. So if you're unchecking this, does it mean we do accept refactoring changes?

Reply:
Updated.